xml/System.IO.Hashing/XxHash32.xml

```diff
@@ -252,12 +265,13 @@
         <Parameter Name="seed" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <param name="seed">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The data to hash.</param>
+        <param name="destination">The buffer that receives the computed hash value.</param>
+        <param name="bytesWritten">On success, receives the number of bytes written to <paramref name="destination" />.</param>
```

Comment:
@gewarren since this is a `Try` method, I wonder if the value could end up being greater than 0 in `bytesWritten` even if the method returns `false`. Hence why they mentioned `On success`, referring to the fact that the `bytesWritten` value can only be trusted if the method returns `true`.

Reply:
I guess it's the expected behavior. I'll apply it.